sky/packages/updater/lib/main.dart

```diff
@@ -67,14 +100,15 @@ class UpdateTask {
   String _dataDir;
   String _tempPath;
 
-  _readLocalManifest() async {
-    String manifestPath = path.join(_dataDir, kManifestFile);
-    String manifestData = await new File(manifestPath).readAsString();
-    _currentManifest = yaml.loadYaml(manifestData, sourceUrl: manifestPath);
+  Future _readLocalManifest() async {
+    String bundlePath = path.join(_dataDir, kBundleFile);
```

Comment:
I'm not convinced that's a likely danger, but I suspect nothing I say will convince you either :). I'll add final.